src/test/java/io/logz/log4j2/OnDiskBuffer.java

```diff
@@ -0,0 +1,40 @@
+package io.logz.log4j2;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class OnDiskBuffer extends Log4j2AppenderTest {
```

Comment:
append "Test" to class name

Reply:
@asafm Going to use Parameterized instead

Reply:
Just make sure it's really easier to read in that form as I found it hard to understand.